src/resolve.ts

```diff
@@ -46,3 +46,11 @@ export function initLayerResolve(resolve: ResolveMapping): ResolveMapping {
 
   return out;
 }
+
+export function initLayerResolve(resolve: ResolveMapping): ResolveMapping {
+  return initResolve(resolve, channel => 'shared');
+}
+
+export function initRepeatResolve(resolve: ResolveMapping): ResolveMapping {
```

Comment:
Yeah, but that can come later. Basically, I will add an argument to pass in the channels.